packages/ui/src/theme/platforms/react-native/transforms.ts

```diff
@@ -0,0 +1,102 @@
+import { DesignToken, Transform } from 'style-dictionary';
+import {
+  isBorderWidth,
+  isFontSize,
+  isFontWeight,
+  isOpacity,
+  isRadius,
+  isShadow,
+  isSpace,
+  isTime,
+} from './matchers';
+
+/*
+TODO: * remove transitionDuration - not supported in RN
+      * shadows
+      * inherit
+      * auto
+      * padding
+      * gap
+      * boxShadow
+      * remove any units (px, rem, em)
+*/
+
+export const ReactNativeTransforms: Record<string, Transform> = {
+  RNFontWeight: {
+    type: 'value',
```

Comment:
Just curious, why do we have to specify `type: 'value'` for each of these transforms?

Reply:
https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/theme/tokens/types/designToken.ts#L20